cfme/tests/ansible/test_embedded_ansible_manual.py

```diff
@@ -0,0 +1,1211 @@
+import pytest
+
+from cfme import test_requirements
+
+
+@pytest.mark.manual
+@test_requirements.ansible
+def test_embed_tower_exec_play_against_vmware():
```

Comment:
@sbulage 
Please add ```tier``` for this test case.
Also take a look at other tests and update with ```@pytest.mark.tier()```

Reply:
@ganeshhubale Added.